script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -99,8 +105,10 @@ begin
       if not (Pos('Git\cmd', ExtractFilePath(ExecStdOut)) = 0) then begin
         // Proxy Git path detected
         Result := ExpandConstant('{pf}');
-      if Is64BitInstallMode then
+      if IsX64 then
         Result := Result + '\Git\mingw64\bin'
+      else if IsARM64 then
+        Result := Result + '\Git\arm64\bin'
```

Comment:
I believe this is supposed to be the path that Git for Windows uses for its binaries.  Just to confirm, Git for WIndows on arm64 uses `arm64\bin`, not `mingw64\bin`?

Reply:
That's correct. [We're working on ARM64 support for Git for Windows](https://github.com/git-for-windows/git/issues/3107) and [it was decided to use `arm64` as the folder name](https://github.com/git-for-windows/MINGW-packages/pull/44/files#diff-df00ed51839a39d696b0022219f61624cc1003df1244dc646f88ae7499f941a0R693) indeed 😊

Reply:
But to be honest, I'm not sure whether this logic is ideal in the first place. E.g. a user can install x86 Git for Windows on a x64 Windows machine, so this logic would fail in that case. When that happens, a user will only have `\Git\mingw32\bin` on their x64 machine then. On top of this function a comment says `// Currently only used to support running the uninstaller for the old Git LFS version.` - [the commit was added 6 years ago](https://github.com/git-lfs/git-lfs/commit/1651e55ce5cc37c69bf5cb3bf92e626b83ad5e0a). I'm not familiar with Git LFS' codebase, but my gut says this code might no longer be needed in the first place. I'll leave that up to you to decide!

Reply:
I also agree that this might no longer be used, but I'm happy to leave it as it is for now.  The InnoSetup stuff is unfamiliar to me as a non-Windows developer, so I'm going to defer to you, since you seem to be more familiar with it.